databases/week3/meal_sharing_additional_queries.sql

```diff
@@ -0,0 +1,51 @@
+-- ADDITIONAL QUERIES
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT meal.id AS 'Meal ID', meal.title AS 'Meal Name', meal.price AS 'Price'
+FROM meal
+WHERE meal.price < 200; 
+
+-- Get meals that still has available reservations
+SELECT meal.id AS 'Meal ID', meal.title AS 'Meal Name', meal.meal_date_time AS 'Meal Date and Time'
+FROM meal
+WHERE NOT EXISTS (
+SELECT * FROM reservation
+WHERE meal.id = reservation.meal_id);
+
+-- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+SELECT meal.id AS 'Meal ID', meal.title AS 'Meal Name'
+FROM meal
+WHERE title like '%Indian%';
+
+-- Get meals that has been created between two dates
+SELECT *
+FROM meal
+WHERE created_date between '2023-01-10' and '2023-01-20';
+
+-- Get only specific number of meals fx return only 5 meals
+SELECT *
+FROM meal
+LIMIT 5;
+
+-- Get the meals that have good reviews
+SELECT meal.id AS 'Meal ID', meal.title AS 'Meal Name', review.title AS 'Review Title', review.description AS 'Review', review.stars AS 'Stars'
+FROM meal
+INNER JOIN review
+ON meal.id = review.meal_id
+WHERE review.stars >= 3;
+	-- Do we need to create a seperate table stars with column star.id and star.name (Bad, Average, Good, Very Good, Excellent)
+
```

Comment:
No, you just decide the ranges of given starts to categorize into` BAD, AVERAGE & GOOD`. for instance when inserting review data, the start for you inserted might be between 1-50, then you can say
bad (1-15) , average(16-30) & good > 30

Reply:
Ok